packages/blockchain-utils-validation/jest.config.json

```diff
@@ -1,9 +1,12 @@
 {
-  "moduleFileExtensions": ["js", "json", "ts"],
-  "testRegex": ".(spec|test).ts$",
   "testEnvironment": "node",
+  "resolver": "jest-resolver-enhanced",
+  "testMatch": ["**/?(*.)+(spec|test).[jt]s?(x)"],
+  "extensionsToTreatAsEsm": [".ts"],
   "moduleNameMapper": {
-    "^@zondax/filecoin-signing-tools/nodejs$": "@zondax/filecoin-signing-tools"
+    "^(\\.{1,2}/.*)\\.js$": "$1"
```

Comment:
man I hate regexes.  What does this do?  I wish you could add a comment explaining this, but I'm guessing comments aren't allowed here because JSON :sob: